content/download.md

```diff
@@ -26,64 +26,67 @@ limitations under the License.
 {% endcomment %}{% endif %}{% comment %}
-{% endcomment %}<a href="{{ site.baseurl }}/history.html#{{ post.tag }}">{{ post.version }}</a>{% comment %}
+{% endcomment %} <a href="{{ post.url }}">{{ post.version }}</a>{% comment %}
 {% endcomment %} | {{ post.date | date_to_string }}{% comment %}
-{% endcomment %} | <a href="https://github.com/apache/{{ site.data.project.unix_name }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
-{% endcomment %} | <a href="{{ p }}/{{ v }}-src.tar.gz{{ q }}">tar</a>{% comment %}
+{% endcomment %} | <a href="{{ site.data.project.source_repository_mirror }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
+// Currently the mirrors have no signitures, so use dist.apache.org:
+{% endcomment %} | {% if post.filename %} <a href="{{ p }}/{{ post.filename }}{{ q }}">download</a>{% comment %}  
+{% endcomment %} , [signatures](https://dist.apache.org/repos/dist/release/{{ site.data.project.incubator_slash_name }}/{{ v }}){% comment %}
+ // currently, we only have the zip file, bring this back when you need it:
+xx endcomment xx {% else %}{% comment %}
+{% endcomment %} <a href="{{ p }}/{{ v }}-src.tar.gz{{ q }}">tar</a>{% comment %}
 {% endcomment %} (<a href="{{ d }}/{{ site.data.project.unix_name }}/{{ v }}/{{ v }}-src.tar.gz.md5">md5</a>{% comment %}
 {% endcomment %} <a href="{{ d }}/{{ site.data.project.unix_name }}/{{ v }}/{{ v }}-src.tar.gz.asc">pgp</a>){% comment %}
 {% endcomment %} {% raw %}<br>{% endraw %}{% comment %}
 {% endcomment %} <a href="{{ p }}/{{ v }}-src.zip{{ q }}">zip</a>{% comment %}
 {% endcomment %} (<a href="{{ d }}/{{ site.data.project.unix_name }}/{{ v }}/{{ v }}-src.zip.md5">md5</a>{% comment %}
 {% endcomment %} <a href="{{ d }}/{{ site.data.project.unix_name }}/{{ v }}/{{ v }}-src.zip.asc">pgp</a>){% comment %}
-{% endcomment %}
+{% endcomment %} xx remove to the left to uncomment {% endcomment %}{% endif %}{% comment %}
+{% endcomment %} | [Release notes]({{ post.releaseNotes }})
 {% endfor %}
-
-Choose a source distribution in either *tar* or *zip* format,
+<br>
+Choose a source distribution in <!-- either *tar* or --> *zip* format,
 and [verify](http://www.apache.org/dyn/closer.cgi#verify)
 using the corresponding *pgp* signature (using the committer file in
-[KEYS](http://www.apache.org/dist/{{ site.data.project.unix_name }}/KEYS)).
+[KEYS](http://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/KEYS)).
 If you cannot do that, the *md5* hash file may be used to check that the
 download has completed OK.
 
+
 For fast downloads, current source distributions are hosted on mirror servers;
-older source distributions are in the
+
+<!-- older source distributions are in the
 [archive](http://archive.apache.org/dist/incubator/{{ site.data.project.unix_name }}/).
+--> 
 If a download from a mirror fails, retry, and the second download will likely
 succeed.
 
+
 For security, hash and signature files are always hosted at
 [Apache](https://www.apache.org/dist).
```

Comment:
Use http://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/ as URL, not the generic Apache URL, as it is hard to navigate to the relevant Rya directory